unime/src/routes/(app)/scan/+page@.svelte

```diff
@@ -126,72 +128,80 @@
 <div class="content-height flex flex-col items-stretch">
   <div class="hide-scrollbar grow overflow-x-hidden overflow-y-scroll">
     <div class="flex h-full w-full flex-col">
-      <!-- visible when NOT scanning -->
-      <div
-        class:invisible={scanning}
-        class="relative flex h-full flex-col items-center justify-center space-y-4 bg-silver p-8 dark:bg-navy"
-      >
-        {#if permissions_nullable && permissions_nullable !== 'granted'}
-          <div class="flex w-3/4 flex-col space-y-4">
-            <div class="flex flex-col items-center rounded-lg bg-rose-100 px-8 py-4 text-rose-500">
-              <CameraSlash class="m-2 h-8 w-8" />
-              <p class="text-center text-[13px]/[24px] font-semibold">{$LL.SCAN.PERMISSION_DENIED()}</p>
+      {#if !scanning && !loading}
+        <!-- This part is only visible when no scanning or loading is happening.
+          Only visible when user has not granted permissions to the camera. -->
+        <div class="relative flex h-full flex-col items-center justify-center space-y-4 bg-silver p-8 dark:bg-navy">
```

Comment:
Might be nitpicking here, but there is usually no reason to use Tailwind's space utilities in combination with a Flex layout. Tailwind has `gap`, `gap-x` and `gap-y`, which corresponds to standard CSS.